test/intercept/https/https.request.test.ts

```diff
@@ -20,37 +36,40 @@ afterEach(() => {
   pool = []
 })
 
-afterAll(() => {
+afterAll(async () => {
   requestInterceptor.restore()
+  await server.close()
 })
 
 test('intercepts an HTTPS GET request', async () => {
   const request = await prepare(
-    httpsRequest('https://httpbin.org/get?userId=123', {
+    httpsRequest(server.makeHttpsUrl('/user?id=123'), {
```

Comment:
I think is better to create a request like `/user/123` instead of `/user?id=123`

Reply:
If I recall correctly, the query parameter is there to test if the request URL gets parsed correctly into query parameters. In case of `/user/123` it becomes a part of a regular URL, and it drops the assertion of query parameters that way.

Reply:
Ah yes i didn't think about it. I thought only the form. Ignore it 🤣